frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/advanced-tab/ssh/SSHAdvancedTab.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react';
+import SSHCreateService from '../../../../ssh-service/SSHCreateService/SSHCreateService';
+import { Stack, StackItem } from '@patternfly/react-core';
+
+const SSHAdvancedTab = () => {
+  return (
+    <Stack hasGutter>
```

Comment:
Why do we need a `Stack` with only one `StackItem`?

Reply:
In 4.9 this file will get much bigger and with more content so its ready for it